web/styles/dark_theme.css

```diff
@@ -349,7 +349,7 @@
     .message_embed .data-container::after {
         background: linear-gradient(
             0deg,
-            hsl(212deg 28% 18%),
+            var(--color-background-stream-message-content),
```

Comment:
Can you remove this now that it's using a variable? It should be the same for dark and light mode now, right?

Reply:
Yes you are absolutely right! Since we are using a variable it will assume the value for the dark theme background. 

My last commit addresses that and the mentions, for both group and direct mentions, color for the fade.